orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/BuildService.groovy

```diff
@@ -33,27 +37,39 @@ class BuildService {
     return UriUtils.encodeFragment(uri.toString(), "UTF-8")
   }
 
-  String build(String master, String jobName, Map<String, String> queryParams) {
-    return igorService.build(master, encode(jobName), queryParams, "")
+  String build(String master, String jobName, Map<String, String> queryParams, String user = null) {
+    return propagateAuth({ igorService.build(master, encode(jobName), queryParams, "") }, user)
```

Comment:
Isn't the auth already propagated to these callers (they're just called from regular orca tasks right?)

Reply:
You're right. I've been able to simplify this a lot now, thanks!